src/SonarScanner.MSBuild.PostProcessor/Resources.resx

```diff
@@ -187,4 +187,7 @@ Config file: {5}</value>
   <data name="WARN_CannotProcessCoverage" xml:space="preserve">
     <value>Cannot process the code coverage information. Look at the previous warnings for details.</value>
   </data>
+  <data name="WARN_UI_MultifileAnalysisEnabled" xml:space="preserve">
+    <value>Multi-file Analysis is enabled. If this was not intended, please set "/d:sonar.scanner.multiFileAnalysis=false" in the begin step.</value>
```

Comment:
I think the term "Multi-file Analysis" isn't self-explanatory. We should also provide a link to the documentation here. @costin-zaharia-sonarsource, what do you think? We can create an issue about adding some documentation links because we still need to write something.

Reply:
On the one hand, I agree.
On the other hand, we ask the user to set `"/d:sonar.scanner.multiFileAnalysis=false"`, which will be a documented parameter that the user can search in the docs.

Do we specify a document URL in other places?